src/applications/vaos/new-appointment/newAppointmentFlow.js

```diff
@@ -358,162 +158,242 @@ const flow = {
+            dispatch(updateFacilityType(FACILITY_TYPES.COMMUNITY_CARE));
+            dispatch(startRequestAppointmentFlow(true));
+            return 'ccRequestDateTime';
+          }
+          if (isEligible) {
+            return 'typeOfFacility';
+          }
+          if (isPodiatry(state)) {
+            // If no CC enabled systems and toc is podiatry, show modal
+            dispatch(showPodiatryAppointmentUnavailableModal());
+            return 'typeOfCare';
+          }
+        }
+
+        dispatch(updateFacilityType(FACILITY_TYPES.VAMC));
+        return VA_FACILITY_V2_KEY;
+      },
     },
     typeOfEyeCare: {
-      ...flow.typeOfEyeCare,
-      url: featureBreadcrumbUrlUpdate
-        ? 'eye-care'
-        : '/new-appointment/choose-eye-care',
+      url: 'eye-care',
+      label: 'Choose the type of eye care you need',
+      async next(state, dispatch) {
+        const data = getFormData(state);
+
+        // check that the result does have a ccId
+        if (getTypeOfCare(data)?.ccId) {
+          const isEligible = await dispatch(checkCommunityCareEligibility());
+
+          if (isEligible) {
+            return 'typeOfFacility';
+          }
+        }
+
+        dispatch(updateFacilityType(FACILITY_TYPES.VAMC));
+        return VA_FACILITY_V2_KEY;
+      },
     },
     typeOfFacility: {
-      ...flow.typeOfFacility,
-      url: featureBreadcrumbUrlUpdate
-        ? 'facility-type'
-        : '/new-appointment/choose-facility-type',
+      url: 'facility-type',
+      label: 'Where do you prefer to receive care?',
+      next(state, dispatch) {
+        if (isCCAudiology(state)) {
+          return 'audiologyCareType';
+        }
+
+        if (isCCFacility(state)) {
+          dispatch(startRequestAppointmentFlow(true));
+          return 'ccRequestDateTime';
+        }
+
+        return VA_FACILITY_V2_KEY;
+      },
     },
     typeOfSleepCare: {
-      ...flow.typeOfSleepCare,
-      url: featureBreadcrumbUrlUpdate
-        ? 'sleep-care'
-        : '/new-appointment/choose-sleep-care',
-    },
-    vaccineFlow: {
-      ...flow.vaccineFlow,
-      url: featureBreadcrumbUrlUpdate
-        ? // IMPORTANT!!!
-          // The trailing slash is needed for going back to the previous page to work properly.
-          // The training slash indicates that 'new-covid-19-vaccine-appointment' is a parent path
-          // with children.
-          //
-          // Ex. /schedule/new-covid-19-vaccine-appointment/
-          //
-          // Leaving the '/' off makes '/schedule' the parent.
-          'covid-vaccine/'
-        : '/new-covid-19-vaccine-appointment',
+      url: 'sleep-care',
+      label: 'Choose the type of sleep care you need',
+      next: VA_FACILITY_V2_KEY,
     },
     vaFacility: {
-      ...flow.vaFacility,
-      url: featureBreadcrumbUrlUpdate
-        ? 'va-facility'
-        : '/new-appointment/va-facility',
+      url: 'va-facility',
+      next: vaFacilityNext,
     },
```

Comment:
Looks like `vaFacility` property is no longer used therefore can be deleted from the flow object. 
It is `vaFacilityV2` is actively being used.